pinot-core/src/main/java/org/apache/pinot/core/segment/index/loader/defaultcolumn/BaseDefaultColumnHandler.java

```diff
@@ -303,6 +304,10 @@ protected void createColumnV1Indices(String column)
         dictionaryElementSize = StringUtil.encodeUtf8(stringDefaultValue).length;
         sortedArray = new String[]{stringDefaultValue};
         break;
+      case BYTES:
+        Preconditions.checkState(defaultValue instanceof byte[]);
```

Comment:
We should probably check if the default value byte array's length should be >= 1?